apps/src/templates/levelSummary/FreeResponseAiSummaryBox.tsx

```diff
@@ -0,0 +1,118 @@
+  // TO DO: Update logic for the proficiency label.  Should be > 20% of students have a proficient answer
+  const aiSummaryTag = (proficiencyCount: number) => (
+    <Tags
+      tagsList={[
+        {
+          label: proficiencyCount > 0 ? 'Proficient' : 'Needs Review',
+          icon: {
+            iconName:
+              proficiencyCount > 0 ? 'circle-check' : 'circle-exclamation',
+            iconStyle: 'solid',
+            title: 'check',
+            placement: 'left',
+          },
+        },
+      ]}
+      size="l"
+      className={
+        proficiencyCount > 0 ? styles.proficientTag : styles.needsReviewTag
+      }
+    />
+  );
+
+  const aiSummaryMessage = (proficiencyCount: number) => (
+    <BodyTwoText>
+      <strong>Reasoning: </strong>
+      {proficiencyCount > 0
+        ? 'This is proficient because more than 1 student has a proficient answer'
+        : 'This is needs review because more than one student needed review'}
+    </BodyTwoText>
+  );
+
+  const countEvaluationsByType = (
+    evaluations: StudentWorkEvaluation[],
+    types: StudentWorkEvaluation['aiEvaluation'][]
+  ): number => {
+    return evaluations.filter(evaluation =>
+      types.includes(evaluation.aiEvaluation)
+    ).length;
+  };
+
+  const proficientStudentCount = studentWorkEvaluations
+    ? countEvaluationsByType(studentWorkEvaluations, ['great', 'ok'])
+    : 0;
+
+  const needsRevisionStudentCount = studentWorkEvaluations
+    ? countEvaluationsByType(studentWorkEvaluations, ['needs revision'])
+    : 0;
+
+  // TO DO: Update this with perhaps different logic...
+  const flaggedStudentCount = studentWorkEvaluations
+    ? countEvaluationsByType(studentWorkEvaluations, ['Cant Evaluate'])
+    : 0;
+  const noResponseStudentCount = studentWorkEvaluations
+    ? countEvaluationsByType(studentWorkEvaluations, ['No Response'])
+    : 0;
+
+  const showEvaluationSummary = studentWorkEvaluations && evaluationComplete;
+
+  // TO DO: Get total student count loaded in
+  return (
+    <div className={styles.aiSummaryContainer}>
+      <div className={styles.leftSide}>
+        <img src={aiBot} alt="Ai Bot" className={styles.botImage} />
+        <Button
+          text="Evaluate student responses"
+          onClick={aiEvaluationHandler}
+          size={'s'}
+          color={'gray'}
+          disabled={disabled}
+          type="secondary"
+          isPending={isPending}
+        />
+      </div>
+      <div className={styles.rightSide}>
+        {showEvaluationSummary && aiSummaryTag(proficientStudentCount)}
+        {showEvaluationSummary && aiSummaryMessage(proficientStudentCount)}
+        {showEvaluationSummary && (
+          <FreeResponseSummaryDataBox
+            totalStudentCount={10}
+            proficientStudentCount={proficientStudentCount}
+            needsRevisionStudentCount={needsRevisionStudentCount}
+            flaggedStudentCount={flaggedStudentCount}
+            noResponseStudentCount={noResponseStudentCount}
```

Comment:
since these are all so similar, I wonder about passing an object of counts as a single prop where 
`counts = {total: 13, proficient: 8, needsRevision: 1, flagged: 1, noResponse: 3} ` It feels more concise to me and may be slightly easier to remove/add if the count categories change but totally a personal preference. Your call - explicit is good too!

Reply:
Yeah, I am kind of thinking that too... I need to still figure out how to get the right flagged students and noResponse students in there correctly.  I think the solution to that might inform this decision in the end.